src/helpers.js

```diff
@@ -157,3 +176,39 @@ export function getCenter(a, b) {
     y: (b.y + a.y) / 2,
   }
 }
+
+function rotate(x, y, xm, ym, a) {
+  var cos = Math.cos,
```

Comment:
Also please use `const` here and create a new variable for `a` (rotationInRadians maybe) to avoid overwriting to function arguments.

Reply:
Sure, I was being lazy and had just copied in something I found online, will rewrite to be more descriptive in a couple of days